script/nbb_tests.clj

```diff
@@ -114,6 +114,24 @@
     (is (thrown? Exception
                  (nbb {:dir "test-scripts/paths-test"} "src/project_dir_not_on_classpath.cljs")))))
 
+(deftest invoked-file-test
+  (testing "calling as a script"
+    (is (= :invoked
+           (nbb
+            {:dir "test-scripts/invoked-file-test"}
+            "src/script.cljs"))))
+  (testing "calling with -m"
+    (is (= :invoked
+           (nbb {:dir "test-scripts/invoked-file-test"}
+                "-m" "core"))))
+  (testing "calling with -e"
+    (is (= :not-invoked
+           (nbb {:dir "test-scripts/invoked-file-test"}
+                "-e" "(require [nbb.core :refer [*file* invoked-file])
+(if (= *file* (invoked-file))
```

Comment:
This code doesn't run in nbb but in babashka, hence this part doesn't really make sense here, I think?

Reply:
That should run in nbb.  It's a string passed to `nbb`